### PR TITLE
l10n: Improve spelling stylistics

### PR DESCRIPTION
**Pull request by Valdnet**

### CODE REVIEW

File: templates/settings-admin.php [hunk 1]
```diff
@@ -32,25 +32,25 @@
...
 			<input id="password-policy-min-length" type="number" value="<?php p($_['minLength']) ?>" />
-			<span><?php p($l->t('Minimal length')) ?></span>
+			<span><?php p($l->t('Minimum password length')) ?></span>
 		</label>
 	</p>
 	<p>
 		<label class="password-policy-number-option">
 			<input id="password-policy-history-size" type="number" value="<?php p($_['historySize']) ?>" />
-			<span><?php p($l->t('User password history')) ?></span>
+			<span><?php p($l->t('User passwords history')) ?></span>
```
Comment (blizzz) [inline templates/settings-admin.php:41]: the singular use of "password" is actually correct here

**Review by blizzz — CHANGES_REQUESTED** [review]
one remark, otherwise :+1:

File: templates/settings-admin.php (hunk 1 above)
Comment (Valdnet) [reply templates/settings-admin.php]: Corrected 😉.

**Review by blizzz — APPROVED** [review]

File: templates/settings-admin.php (hunk 1 above)
Comment (blizzz) [reply templates/settings-admin.php]: Thank you :)

**Pull request merged**